src/main/java/org/jahia/modules/serversettings/portlets/AssemblerTask.java

```diff
@@ -76,50 +75,30 @@ public AssemblerTask(File tempDir, File webapp) {
 
     public File execute() throws Exception {
         long timer = System.currentTimeMillis();
-        logger.info("Got a command to prepare " + getWebapp() + " WAR file to be deployed into the Pluto container");
+        logger.info("Got a command to prepare {} WAR file to be deployed into the Pluto container", getWebapp());
         validateArgs();
 
-        BasePortletHelper portletHelper = getPortletHelperInstance();
```

Comment:
Just curious why the portlet helper code is being removed. Didn't see any mention of it in the story.

Reply:
It looks like this PR could be a dupe of this one? https://github.com/Jahia/serverSettings/pull/99

Reply:
Indeed, I did the changes in parallel, but created two PRs thinking that they were on different files.